src/components/filterslider.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react'
+import {clothingCategory,stylesByCategory,clothingColor,clothingSeason} from '../containers/api'
+import  '../styles/filterslider.css'
+
+
+export default class Filter extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            categories: clothingCategory,
+            chosencategory: '',
+            styles: [],
+            chosenstyle: '',
+            colors: clothingColor,
+            chosencolor: '',
+            seasons: clothingSeason,
+            chosenseason: ''
+        }
+    }
+
+
+    //---------HANDLES SELECTIONS 
+    handleCategory = (e) => {
+      this.setState({
+          chosencategory: e.target.value,
+          styles: stylesByCategory[e.target.value]
+        })
+      
+}
+
+    handleStyle = (e) => {
+        console.log('STYLE selected', e.target.value);
+        this.setState({ chosenstyle: e.target.value })
+
+        
+    }
+
+    handleColor = (e) => {
+        console.log('COLOR selected', e.target.value);
+        this.setState({ chosencolor: e.target.value })
+    }
+
+    handleSeason = (e) => {
+        console.log('SEASON selected', e.target.value);
+        this.setState({ chosenseason: e.target.value })
+    }
+    
+    //-------------------------------------------
+    render() {
+        console.log('category list', this.state.category)
+        console.log('style list', this.state.style)
+        console.log('season list', this.state.seasons)
+        const { categories, styles } = this.state
```

Comment:
I don't think you need to grab these values from component state. `categories` is the same as the const `clothingCategories` which you import at the top of the file.
The values for `styles` can change depending on user entry, but here's what I would do instead:
```
const { chosencategory } = this.state; // get the user's selected category from state; it could be ''
const styleOptions = clothingStyles[chosencategory] || []; // if chosencategory is `''` you'll get `[]`
```